setup.py

```diff
@@ -50,7 +50,6 @@ def find_scripts(script_dir, prefix):
         'scipy',
         'seaborn',
         'tables',
-        'joblib',
```

Comment:
NO! please list everything, that you directly import here. You should not rely on transitive dependencies from ctapipe.

If you have `import package` in lstchain, <package> belongs into the setup.py